code/game/objects/items/weapons/RCD.dm

```diff
@@ -125,8 +125,8 @@ obj/item/weapon/construction
 	var/canRturf = 0
 	var/ranged = FALSE
 	var/airlock_type = /obj/machinery/door/airlock
+	var/airlock_glass = FALSE // So the floor's rcd_act knows how much ammo to use
```

Comment:
I don't really like the method I had to use for this. 
As I saw it, I either had to do:
1. This.
2. Put a monstrous is check for every glass airlock type into floor.dm
3. Repath the doors to /glass/ and update every single map accordingly

Can a maintainer chime in on if this is acceptable?